src/target/stm32mp15.c

```diff
@@ -85,12 +85,23 @@ static void stm32mp15_detach(target_s *target);
 static bool stm32mp15_ident(target_s *const target, const bool cortexm)
 {
 	const adiv5_access_port_s *const ap = cortex_ap(target);
+	DEBUG_TARGET("%s: AP partno %X, part_id %X\n", __func__, ap->partno, target->part_id);
```

Comment:
This and the block below get similar comments to their counterparts in the STM32H7 support, rewording accordingly for MP15x instead of H7.